assets/_scss/elements/_buttons.scss

```diff
@@ -103,40 +103,36 @@ button,
 
   &.usa-button-outline {
     background-color: $color-white;
-    border: 2px solid;
-    border-color: $color-primary;
+    box-shadow: inset 0 0 0 2px $color-primary;
     color: $color-primary;
-    padding: .8rem 1.8rem;
 
     &:hover,
     &.usa-button-hover {
-      border-color: $color-primary-darker;
+      box-shadow: inset 0 0 0 2px $color-primary-darker;
```

Comment:
Is there a way to store `inset 0 0 0 2px` in a variable to use in all the places rather the repeating it?